### DISCUSSION TITLE
Dockerhub is missing v1.13.1 of nginxproxy/acme-companion

### DISCUSSION THREAD

**Discussion by te-online** [nginx-proxy/acme-companion#831] [Q&A] [answered]
Bug description
-----------------

Neither `jrcs/letsencrypt-nginx-proxy-companion` (seems to have disappeared entirely) nor `nginxproxy/acme-companion` seem to have a version 1.3.1 available that is mentioned in the top section of the Readme:

> The last tagged version that uses simp_le is v1.13.1.

For compatibility reasons I'd like to try out this version, but don't know how to reference it. Did this build get removed entirely, so that I'm forced to migrate to v2.x.x right now?

Errors look like this

> Pulling nginx-proxy-letsencrypt (jrcs/letsencrypt-nginx-proxy-companion:1.13.1)...
> ERROR: manifest for jrcs/letsencrypt-nginx-proxy-companion:1.13.1 not found: manifest unknown: manifest unknown

> Pulling nginx-proxy-letsencrypt (nginxproxy/acme-companion:1.13.1)...
> ERROR: manifest for nginxproxy/acme-companion:1.13.1 not found: manifest unknown: manifest unknown

**Comment by buchdag** [comment, marked as answer]
> Neither `jrcs/letsencrypt-nginx-proxy-companion` (seems to have disappeared entirely)

The small tricky bit here is that we went from `vx.x.x` (prior to version `2`) to just `x.x.x`), so you have to do

```console
docker pull jrcs/letsencrypt-nginx-proxy-companion:v1.13.1
```

 [The image is still there](https://hub.docker.com/layers/jrcs/letsencrypt-nginx-proxy-companion/v1.13.1/images/sha256-9cc446c73952544fea95086bc762e5b64f480d30abde886ddb2212825a64905e?context=explore).

**Reply by buchdag** [reply]
I also just pushed `v1.13.1` / `v1.13`  to `nginxproxy/acme-companion`.

So either

```console
docker pull nginxproxy/acme-companion:v1.13.1
```

or

```console
docker pull nginxproxy/acme-companion:v1.13
```

**Reply by te-online** [reply]
@buchdag Thanks for your quick response. That's perfect 🎉 
I was trying this: https://hub.docker.com/search?q=jrcs%2Fletsencrypt-nginx-proxy-companion&type=image
But maybe you simply unlisted the old name.

**Reply by buchdag** [reply]
Looks like DockerHub don't like `/` in searches : https://hub.docker.com/search?q=jrcs&type=image
